CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+# Optimizely Csharp SDK Changelog
+
+## 3.2.0
```

Comment:
Version should be 3.3.3 and the date August 22 (aiming to release tomorrow)

Reply:
Should be 3.3.0, right?

Reply:
yeah 3.3.0.